sklearn/linear_model/tests/test_base.py

```diff
@@ -165,6 +211,32 @@ def test_linear_regression_sparse(random_state=0):
         assert_array_almost_equal(ols.predict(X) - y.ravel(), 0)
 
 
+@pytest.mark.parametrize(
+    'normalize, n_warnings, warning',
+    [(True, 1, FutureWarning),
+     (False, 1, FutureWarning),
+     ("deprecated", 0, None)]
+)
+# FIXME remove test in 1.4.0
+def test_assure_warning_when_normalize(normalize, n_warnings, warning):
+    # check that we issue a FutureWarning when normalize was set
+    rng = check_random_state(0)
+    n_samples = 200
+    n_features = 2
```

Comment:
```suggestion
    n_samples, n_features = 200, 2
```

Reply:
honestly, this kind of style suggestion is subjective. @maikia feel free to resolve ;)